showpass/Showpass-E2E-tests
# Add error handling to TA-51

**Description:**
- Added error handling to cypress/e2e/api-requests/transactions-api.cy.js
- Minor tweaking

**How to test:**
Run the following spec and ensure the test passes cypress/e2e/api-requests/transactions-api.cy.js

**Test execution results:**

<img width="869" alt="Tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/491540b9-cd66-4511-b295-42f597a9cb12">
